src/components/MobileProject/MobileProject.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import sampleImg from '../../assets/images/sampleProjectImg.png';
+import {
+  moblieProject,
+  moblieProjectTitle,
+  projectImg,
+} from './MoblileProject.module.scss';
+
+export const MoblieProject = (props) => {
+  return (
+    <>
+      <img src={sampleImg} alt="project image" className={`${projectImg}`} />
```

Comment:
please provide sampleImg using props (so it will be passed in index.js, and there you can import sampleImg

Reply:
dokładnei jest to kluczowe, by to robić propsem z odpowiednim prop-typem, bo inaczej cieżko bedzie nam dodawać zdjęcia do projektów kolejnych